example/variables.tf

```diff
@@ -0,0 +1,30 @@
+
+variable "api_name" {
+  description = "The API Gateway API name, for associating with this custom domain"
+  type        = string
+}
+
+variable "aws_region" {
+  default = "us-east-1"
```

Comment:
I know it is an example, but I thought there was talk about releasing to east-2 by default?